### PR TITLE
Issue #3607: Call layout templates what they are

### LINKED ISSUE #3607: [UX] Disambiguate "Layouts" vs "Layout templates" in the admin interface.
I noticed that the tab on the Layouts page is labeled `Install new Layouts` when it should really be `Install new Layout TEMPLATES` as per our new disambiguation between `Layouts` and `Layout templates`. 
Screen Shot 2019-03-21 at 5 43 32 PM

---
~PR by @seancav03 https://github.com/backdrop/backdrop/pull/2599~
Updated PR https://github.com/backdrop/backdrop/pull/3278 (includes commit above)

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/3607

### CODE REVIEW

File: core/modules/layout/layout.admin.inc [hunk 1]
```diff
@@ -1234,7 +1234,7 @@ function layout_content_form($form, &$form_state, Layout $layout, $renderer_name
 }
 
 /**
- * Validate the layout settings form.
```
Comment (jenlampton) [inline core/modules/layout/layout.admin.inc]: @klonos There are other layout settings that we will be adding here. Let's not change all the code comments now, and then need to change them back.

File: core/modules/layout/layout.admin.inc [hunk 2]
```diff
@@ -2930,7 +2930,7 @@ function layout_contexts_form_element(array $available_contexts, array $current_
 /**
  * AJAX handler that opens a form in a dialog.
  *
- * This AJAX handler is used both on the layout settings form and the layout
```
Comment (jenlampton) [inline core/modules/layout/layout.admin.inc]: revert this?

File: core/modules/layout/layout.admin.inc [hunk 3]
```diff
@@ -3127,7 +3127,7 @@ function layout_ajax_form_update($form, $form_state) {
 }
 
 /**
- * Administrative form for layout settings.
```
Comment (jenlampton) [inline core/modules/layout/layout.admin.inc]: revert this?

File: core/modules/layout/layout.admin.inc [hunk 4]
```diff
@@ -3191,7 +3191,7 @@ function layout_settings_page($form, &$form_state) {
 
 
 /**
- * Validation for layout settings page.
```
Comment (jenlampton) [inline core/modules/layout/layout.admin.inc:3194]: revert this?

**Review by yorkshire-pudding — COMMENTED** [review]
Looks good to me

**Comment by ghost** [comment]
Thanks, merged into 1.x and 1.24.x.

**Pull request merged**